### PR TITLE
fix(auth-model): changes made for model to be responsive in store

### LINKED ISSUE #2797: [BUG] Auth models aren't responsive in refine store
### Describe the bug

**In desktop view,**
1. Sign in model close button is visible only half.

**In mobile view,**
1. Both sign in and login models are cut on top and bottom.
2. Close button is not visible.

### Steps To Reproduce

1. Go to https://store.refine.dev/
2. Click on auth button
3. Click on sign up button
4. Close button of model is not visible(model not responsive)
5. In mobile view also models are not responsive

### Expected behavior

Auth models should be responsive

### Screenshot

image
image

### Desktop

_No response_

### Mobile

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by s-vamshi**
Made necessary changes to make auth model responsive
This is how it looks now - 
image

### Closing issues

closes #2797 

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by s-vamshi** [comment]
@omeraplak can you please review this PR when you are free?

File: examples/store/src/components/ui/Modal/Modal.module.css [hunk 1]
```diff
@@ -1,17 +1,17 @@
 .root {
-  @apply fixed bg-black bg-opacity-40 flex items-center inset-0 z-50 justify-center;
+  @apply fixed bg-black bg-opacity-40 grid overflow-y-scroll flex items-center inset-0 z-50 justify-center;
```
Comment (aliemir) [inline examples/store/src/components/ui/Modal/Modal.module.css:2]: isn't `grid` and `flex` classes conflict with each other? @s-vamshi

Comment (s-vamshi) [reply examples/store/src/components/ui/Modal/Modal.module.css]: Yes they will conflict and i have removed it. Can you please check?

**Comment by s-vamshi** [comment]
@omeraplak can you please review my PR?

File: examples/store/src/components/ui/Modal/Modal.module.css [hunk 2]
```diff
@@ -1,17 +1,17 @@
 .root {
-  @apply fixed bg-black bg-opacity-40 flex items-center inset-0 z-50 justify-center;
+  @apply fixed bg-black bg-opacity-40 grid overflow-y-scroll items-center inset-0 z-50 justify-center;
   backdrop-filter: blur(0.8px);
   -webkit-backdrop-filter: blur(0.8px);
 }
 
 .modal {
-  @apply bg-primary p-12 border border-accent-2 relative;
+  @apply rounded bg-primary border border-accent-2 relative;
```
Comment (omeraplak) [inline examples/store/src/components/ui/Modal/Modal.module.css:8]:
```suggestion
  @apply bg-primary border border-accent-2 relative;
```

**Comment by s-vamshi** [comment]
@omeraplak committed changes which you have suggested.

File: examples/store/src/components/ui/Modal/Modal.module.css [hunk 3]
```diff
@@ -1,17 +1,17 @@
 .root {
-  @apply fixed bg-black bg-opacity-40 flex items-center inset-0 z-50 justify-center;
+  @apply fixed bg-black bg-opacity-40 grid overflow-y-scroll items-center inset-0 z-50 justify-center;
```
Comment (salihozdemir) [inline examples/store/src/components/ui/Modal/Modal.module.css:2]:
```suggestion
  @apply fixed bg-black bg-opacity-40 flex overflow-y-scroll items-center inset-0 z-50 justify-center;
```

File: examples/store/src/components/ui/Modal/Modal.module.css [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
   backdrop-filter: blur(0.8px);
   -webkit-backdrop-filter: blur(0.8px);
 }
 
 .modal {
-  @apply bg-primary p-12 border border-accent-2 relative;
+  @apply bg-primary border border-accent-2 relative;
```
Comment (salihozdemir) [inline examples/store/src/components/ui/Modal/Modal.module.css:8]:
```suggestion
  @apply bg-primary border border-accent-2 relative p-2 md:p-12;
```

File: examples/store/src/components/ui/Modal/Modal.module.css [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 }
 
 .modal:focus {
   @apply outline-none;
 }
 
 .close {
-  @apply hover:text-accent-5 transition ease-in-out duration-150 focus:outline-none absolute right-0 top-0 m-6;
+  @apply hover:text-accent-5 transition ease-in-out duration-150 focus:outline-none absolute right-0 top-0 m-4;
```
Comment (salihozdemir) [inline examples/store/src/components/ui/Modal/Modal.module.css:16]:
```suggestion
  @apply hover:text-accent-5 transition ease-in-out duration-150 focus:outline-none absolute right-0 top-0 m-6;
```

**Review by salihozdemir — CHANGES_REQUESTED** [review]
Hey @s-vamshi,
You catch a good point!, But your changes change out design, so i reviewed and changed the code please check it out. 🙏🏻

File: examples/store/src/components/ui/Modal/Modal.module.css (hunk 3 above)
Comment (s-vamshi) [reply examples/store/src/components/ui/Modal/Modal.module.css:2]: @salihozdemir if we go with flex class the problem still exists.
In desktop view, close button is only visible half if we use flex.
And in mobile view, This is how it looks
Even though scroll bar is at the top, model's top portion is not visible.
image

Comment (salihozdemir) [reply examples/store/src/components/ui/Modal/Modal.module.css:2]: @s-vamshi, You are right! 🚀 I missed it!

**Review by salihozdemir — APPROVED** [review]

File: examples/store/src/components/ui/Modal/Modal.module.css (hunk 3 above)
Comment (s-vamshi) [reply examples/store/src/components/ui/Modal/Modal.module.css:2]: That's the reason i have gone with grid.

**Review by aliemir — APPROVED** [review]

**Pull request merged**